### ISSUE TITLE
DNS SERVFAILs are producing malformed packet warnings

### ISSUE THREAD

**Issue by jagerman** [loki-project/loki-network#510]
When doing a DNS lookup, I get a proper SERVFAIL and deferral to the second DNS server, but there seems to be some extra unexpected data in the response from the lokinet resolver:

```
$ dig a example.com @127.3.2.1
;; Warning: Message parser reports malformed message packet.

; <<>> DiG 9.11.5-P4-1-Debian <<>> a example.com @127.3.2.1
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: SERVFAIL, id: 30298
;; flags: qr aa ra ad; QUERY: 1, ANSWER: 1, AUTHORITY: 0, ADDITIONAL: 1
;; WARNING: Message has 24 extra bytes at end

;; QUESTION SECTION:
;example.com.			IN	A

;; Query time: 0 msec
;; SERVER: 127.3.2.1#53(127.3.2.1)
;; WHEN: Fri Apr 12 01:07:59 ADT 2019
;; MSG SIZE  rcvd: 76
```

(Note the initial warning + the WARNING in the middle about 24 extra bytes)

**Comment by majestrate** [comment]
attempted a fix in #511

**Comment by jagerman** [comment]
Confirmed fixed.
